src/libraries/Common/src/Interop/Windows/SspiCli/SecuritySafeHandles.cs

```diff
@@ -310,10 +311,15 @@ public override bool IsInvalid
 
     internal sealed class SafeFreeCredential_SECURITY : SafeFreeCredentials
     {
+#pragma warning disable 0649
+        // This is used only by SslStream but it is included elsewhere
+        public X509Certificate? LocalCertificate;
+#pragma warning restore 0649
         public SafeFreeCredential_SECURITY() : base() { }
 
         protected override bool ReleaseHandle()
         {
+            LocalCertificate?.Dispose();
```

Comment:
Is it possible this might throw? If it does, we'll leak the _handle.

Reply:
same discussion as in https://github.com/dotnet/runtime/pull/92684#discussion_r1446486576